projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -55,7 +55,7 @@
 
 .jetpack-form-file-field__preview {
     align-items: center;
-    animation: show 200ms 100ms cubic-bezier(0.38, 0.97, 0.56, 0.76) forwards;
+    animation: showFileField 300ms 100ms cubic-bezier(0.38, 0.97, 0.56, 0.76) forwards;
```

Comment:
Needs "prefers reduced motion" wrapper to disable the animation.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion